modules/mod_comment/models/m_comment.erl

```diff
@@ -103,9 +106,13 @@ get(CommentId, Context) ->
 
 
 %% @doc Insert a new comment. Fetches the submitter information from the Context.
-%% @todo Insert external ip address and user agent string
 -spec insert(m_rsc:resource(), Name::string(), Email::string(), Message::string(), Is_visible::boolean(), #context{}) -> {ok, pos_integer()} | {error, any()}.
```

Comment:
The spec in line 109 uses a record, please define a type for the record and use that instead.